sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -95,6 +95,10 @@
     private static final String DEFAULT_SWITCH_OFF_IO_THREAD_FOR_RESPONSE_NAME = "COSMOS.SWITCH_OFF_IO_THREAD_FOR_RESPONSE";
     private static final boolean DEFAULT_SWITCH_OFF_IO_THREAD_FOR_RESPONSE = false;
 
+    // OpenConnectionsAndInitCaches Constants
+    private static final String DEFAULT_OPEN_ASYNC_RETRIES_COUNT_NAME = "COSMOS.OPEN_ASYNC_RETRIES_COUNT";
```

Comment:
Shouldn't we call it 
```suggestion
    private static final String DEFAULT_OPEN_CONNECTIONS_RETRIES_COUNT_NAME = "COSMOS.DEFAULT_OPEN_CONNECTIONS_RETRIES_COUNT";
```

Reply:
changed to: 
` private static final String DEFAULT_OPEN_CONNECTIONS_RETRIES_COUNT_NAME = "COSMOS.DEFAULT_OPEN_CONNECTIONS_RETRIES_COUNT";`